archinstall/lib/installer.py

```diff
@@ -332,10 +332,14 @@ def minimal_installation(self):
 			vendor = cpu_vendor()
 			if vendor == "AuthenticAMD":
 				self.base_packages.append("amd-ucode")
+				if (ucode := pathlib.Path(f"{self.target}/boot/amd-ucode.img")).exists():
+					ucode.unlink()
 			elif vendor == "GenuineIntel":
 				self.base_packages.append("intel-ucode")
+				if (ucode := pathlib.Path(f"{self.target}/boot/intel-ucode.img")).exists():
+					ucode.unlink()
 			else:
-				self.log("Unknown cpu vendor not installing ucode")
+				self.log("Unknown cpu vendor not installing ucode", level=logging.INFO)
```

Comment:
This text would be more grammatically correct if it said "Unknown CPU vendor detected; not installing ucode."

Reply:
I'll convert it to a `DEBUG` and correct the grammar.